fitsio/fitsio_pywrap.c

```diff
@@ -40,6 +40,13 @@ struct PyFITSObject {
     fitsfile* fits;
 };
 
+#ifdef FITSIO_PYWRAP_ALWAYS_NONSTANDARD_STRINGS
+static int fits_use_standard_strings(void)
+{
+    return 0;
+}
+#endif
+
```

Comment:
@esheldon Why don't we want this macro around only the return value?

Reply:
Ohhhhh. This function is here in case the version of fitsio doesn't have this symbol. NM.

Reply:
I think that's correct